SwagLyricsBot/general_commands.py

```diff
@@ -106,10 +108,61 @@ async def send_lyrics():
             await log.add_sub_log(f"Error: {ex}", ConsoleColors.FAIL, True)
             print(traceback.print_exception(type(ex), ex, ex.__traceback__))
             log.change_log_success_status(False)
-            await ctx.send(f"There was an error while processing your request. Please try again in a few seconds.")
+            await ctx.send("There was an error while processing your request. Please try again in a few seconds.")
         finally:
             await log.send_webhook()
 
+    @tasks.loop(seconds=5)
+    async def vibe_mode(self):
+        print('this is inside the loop')
+        for user, info in self.current.items():
+            print(user.id, info)
+            channel = self.bot.get_channel(info['channel'])
+            await channel.send('ope')
+            try:
+                song, artists = self.get_spotify_data(user)
+                print(song, artists)
+                if (song, artists) == info['playing']:
+                    print("continuing")
+                    continue
+                # song has changed
+                self.current[user]['playing'] = (song, artists)
+                artists_string = self.artists_to_string(artists)
+                debug_string = f"Getting lyrics for {song} by {artists_string}"
+                # await log.add_sub_log(debug_string)
+                print('sending lyrics bb')
+                await channel.send(debug_string)
+
+                await self.send_lyrics(channel, song, artists)
+            except LyricsError as e:
+                print('No activity detected, stopping loop.')
+                await channel.send("No activity detected, killing the vibe.")
+                del self.current[user]
+                if not self.current:
+                    print('stopping loop')
+                    self.vibe_mode.cancel()
+
+    @commands.command()
+    async def vibe(self, ctx):
+        prev = self.current.copy()
+        # add current user to dict
+        self.current[ctx.author] = {'playing': (None, None),
+                                    'channel': ctx.channel.id}
+        await ctx.send('one vibe mode coming right up.')
+        print(prev)
+        if not prev:  # compare using previous current
+            print("starting loop")
+            self.vibe_mode.start()
+
+    @commands.command()
+    async def kill(self, ctx):
```

Comment:
Maybe rename this to `kill_vibe` or something to prevent ambiguity

Reply:
ambiguity with what?